### PR TITLE
[chore](be) Add default timezone files

### PR DESCRIPTION
**Pull request by zclllyybb**
## Proposed changes

Issue Number: close #xxx

It's about 600kb and 7.5m after unzipping.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

**Review by HappenLee — DISMISSED** [review]
LGTM

File: build.sh [hunk 1]
```diff
@@ -634,6 +634,10 @@ if [[ "${OUTPUT_BE_BINARY}" -eq 1 ]]; then
     cp -r -p "${DORIS_HOME}/be/output/bin"/* "${DORIS_OUTPUT}/be/bin"/
     cp -r -p "${DORIS_HOME}/be/output/conf"/* "${DORIS_OUTPUT}/be/conf"/
     cp -r -p "${DORIS_HOME}/be/output/dict" "${DORIS_OUTPUT}/be/"
+    if [[ ! -d "${DORIS_HOME}/zoneinfo" ]]; then
```
Comment (yiguolei) [inline build.sh:637]: 不要把zoneinfo.tar.gz 放在根目录下，可以建立一个单独的目录 resource，放到 $DORIS_HOME/resource

File: build.sh [hunk 2]
```diff
@@ -634,6 +634,10 @@ if [[ "${OUTPUT_BE_BINARY}" -eq 1 ]]; then
...
+        tar -xzf "${DORIS_HOME}/zoneinfo.tar.gz" -C "${DORIS_HOME}"/
```
Comment (yiguolei) [inline build.sh:638]: 只检查 zoneinfo 这个文件夹是不够的，最好检查其中的一个文件是否存在。否则可能是空文件夹。

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

**Review by BiteTheDDDDt — APPROVED** [review]

**Review by Gabriel39 — APPROVED** [review]
LGTM

**Comment by CalvinKirs** [comment]
I would say no, could you provide a license for this package? Also, we should not put it in the source package, or at least the source package in the released version should not include the binary package.

**Pull request merged**